book/src/move-basics/option.md

```diff
@@ -35,15 +35,16 @@ a missing middle name. Instead, we can use the `Option` type to represent the mi
 {{#include ../../../packages/samples/sources/move-basics/option.move:registry}}
 ```
 
-In the example above, the `middle_name` field is of type `Option<String>`. This means that the
-`middle_name` field can either contain a `String` value or be empty. This makes it clear that the
-middle name is optional, and it avoids the need for extra checks to differentiate between an empty
-string and a missing middle name.
+In the example above, the `middle_name` field is of type `Option<String>`. This means that the middle_name
+field can either contain a String value, wrapped in Some, or be explicitly empty, represented by None. Using
+the Option type makes the optional nature of the field clear, avoiding ambiguity and the need for extra
+checks to differentiate between an empty string and a missing middle name.
 
 ## Using Option
 
-To use the `Option` type, you need to import the `std::option` module and use the `Option` type. You
-can then create an `Option` value using the `some` or `none` methods.
+The Option type, along with the std::option module, is implicitly imported in Move. This means you can use
```

Comment:
```suggestion
The `Option` type, along with the `std::option` module, is implicitly imported in Move. This means you can use
```
Looks like the removal of tics is purposeful but not consistent with rest of page.